algorithms/cql.py

```diff
@@ -1,7 +1,7 @@
 # source: https://github.com/young-geng/CQL/tree/934b0e8354ca431d6c083c4e3a29df88d4b0a24d
 # STRONG UNDER-PERFORMANCE. BUT IN IQL PAPER IT WORKS SOMEHOW
```

Comment:
Is this still true?

Reply:
True for AntMaze medium and large. Locomotion and Maze2d are fine

Reply:
I think we should specify this as usually we are more interested in locomotion results and such comment can scare people away